shared/middleware/tables_api.rb

```diff
@@ -122,7 +122,7 @@ class TablesApi < Sinatra::Base
               "contains a name for each column:<br><br>#{columns.join(',')}"
         halt 400, {}, msg unless column
       end
-      CSV.foreach(tempfile, headers: true) do |row|
+      CSV.foreach(tempfile, headers: true, skip_blanks: true) do |row|
```

Comment:
Trying to understand this reviewing the Ruby [CSV documentation](http://ruby-doc.org/stdlib-1.9.2/libdoc/csv/rdoc/CSV.html).  I find it awesome and very confusing that ruby understands here that `headers: true, skip_blanks: true` are not arguments 2 and 3, but are both part of a hash that makes up argument 2.  But this does [conform to the style guide](https://github.com/bbatsov/ruby-style-guide#no-braces-opts-hash).